cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1806,6 +1806,195 @@ search = | savedsearch "Palo Alto Firewall - Network Compromise - DDoS Attack Pr
+| eval cyences_severity = case(severity=="informational", "info", true(), severity) \
+| rename logdesc as description
+| table _time, subtype, signature, cyences_severity, action, description, dvc \
+| `cs_human_readable_time_format(_time, event_time)` \
+| `cs_fortigate_high_system_alerts_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_fortigate_high_system_alerts_filter
+action.cyences_notable_event_action.contributing_events = `cs_fortigate` sourcetype="fortigate_event" severity IN ("high" "critical") NOT "*license*"
+action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by dvc
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_fortigate` sourcetype="fortigate_event" severity IN ("high" "critical") NOT "*license*" dvc=$row.dvc$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Fortigate
+
+
+# Reports
+[Fortigate Firewall - Network Compromise - DDoS Attack Prevented]
+disabled = 1
+action.email.useNSSubject = 1
+alert.track = 0
+auto_summarize = 1
+auto_summarize.dispatch.earliest_time = -1mon@d
+description = Fortigate Firewall prevented DDoS attack events.\
+\
+Data Collection - Fortigate Networks firewalls threat data. (Fortinet Fortigate Add-on for Splunk)
+search = `cs_fortigate` sourcetype="fortigate_utm" \
+| stats count, latest(_time) as last_seen, dc(src_port) as dc_src_port, dc(dest_ip) as dc_dest_ip, values(signature) as signature, values(dvc) as dvc, values(srccountry) as src_location, dc(dest) as dc_dest, values(action) as action by src | sort - count \
+| `cs_fortigate_ddos_prevented_filter`
+action.cyences_notable_event_action.products = Fortigate
+
+[Fortigate Firewall - Network Compromise - Inbound Traffic from Blocked IPs]
+disabled = 1
+action.email.useNSSubject = 1
+alert.track = 0
+auto_summarize = 1
+auto_summarize.dispatch.earliest_time = -1mon@d
+description = Network traffic coming from blocked IPs. The ip_blocked_list lookup is being used to identify the blocked IPs.\
+\
+Data Collection: Fortigate fortigate_traffic sourcetype data. (Fortinet Fortigate Add-on for Splunk)
+search = `cs_fortigate` sourcetype="fortigate_traffic" action="allowed" `cs_filter_private_ips` packets_in>0 \
+| stats count, latest(_time) as last_seen, values(dvc) as dvc, dc(dvc) as dc_dvc, dc(src_port) as dc_src_port, values(srccountry) as src_location, sum(packets_in) as packets_in, sum(packets_out) as packets_out, dc(dest_ip) as dc_dest_ip, values(dest_port) as dest_port, dc(dstcountry) as dc_dest_location, values(app) as app, values(rule) as firewall_rule by src_ip \
+| `cs_fortigate_search_blocked_ip("src_ip")` | sort -count \
+| `cs_fortigate_blocked_ip_inbound_filter`
+action.cyences_notable_event_action.products = Fortigate
+
+
+[Fortigate Firewall - Network Compromise - Outbound Traffic to Blocked IPs]
```

Comment:
Can we generalize this alert with `Network Traffic` data-model?
- ensure it covers atleast Palo Alto and Fortigate logs?